bigframes/pandas/__init__.py

```diff
@@ -647,6 +648,9 @@ def read_gbq_function(function_name: str):
 close_session = global_session.close_session
 reset_session = global_session.close_session
 
+# SQL Compilation uses recursive algorithms on deep trees
+# 10M tree depth should be sufficient to generate any sql that is under bigquery limit
+sys.setrecursionlimit(10000000)
```

Comment:
Let's get the current so that we aren't decreasing the limit (though I expect that to be unlikely)

Reply:
done, now doing max for 10000000 and current recursion limit